### ISSUE TITLE
Tern on MacOS docker_run.sh command generates empty output.txt

### ISSUE THREAD

**Issue by mkbhanda** [vmware/tern#527]
**Describe the bug**
Tern on MacOS builds but docker_run.sh command generates empty output.txt 

**To Reproduce**
1. git clone https://github.com/vmware/tern.git 
2. docker build -t ternd .
3. ./docker_run.sh workdir ternd "-l report -i debian:buster" > output.txt

Results in an empty output.txt file. 

**Error in terminal**
```
020-01-28 19:25:51,509 - ERROR - rootfs - Command failed. getfattr: ./usr/bin/tee: Operation not sup
11:34

then a whole bunch of these getfattr operation not permitted

getfattr: ./lib/x86_64-linux-gnu/libc-2.28.so: Operation not supported
getfattr: ./run/utmp: Operation not supported
020-01-28 19:25:51,513 - WARNING - report - Error in loading image: Command '['/usr/lib/python3.7/site-packages/tern/tools/fs_hash.sh', '/hostmount/temp/3e5eb770a25e8700e63a79e57e4cb51837589f72410afe73254eccdf19094040/contents']' returned non-zero exit status 1.
2020-01-28 19:25:51,515 - WARNING - run - Cannot retrieve full image metadata
2020-01-28 19:25:51,516 - DEBUG - run - Teardown...
2020-01-28 19:25:51,539 - DEBUG - rootfs - Running command: rm -rf /hostmount/temp/mergedir
2020-01-28 19:25:51,544 - DEBUG - rootfs - Running command: rm -rf /hostmount/temp/workdir
2020-01-28 19:26:01,253 - DEBUG - __main__ - Report completed.
2020-01-28 19:26:01,254 - DEBUG - __main__ - Finished
```

**Expected behavior**
Expected an output.txt file with a BoM of the Debian container.

**Environment you are running Tern on**

Enter all that apply
- Output of 'tern --version'
   git commit sha 0939ffea7ac6b350f2465caee510f4ed2e99e603. (Jan 22, 2020)
[tern_log.txt](https://github.com/vmware/tern/files/4124303/tern_log.txt)

- Operating System: MacOS Mojave 10.14.6 

- Container OS phone: 3.0 (per Dockerfile)
- Python version (3.6 or higher)

**Please attach files if they exist**
- tern.log
- report.*

**Comment by nishakm** [comment]
Confirmed. It looks like some operations of `xattr` are not supported on APFS. In this case, it is easier to run tern using Vagrant or natively on a Linux desktop.

**Comment by rnjudge** [comment]
Not sure if this is explicitly a bug as it is expected behavior per the documentation: "Tern does not run natively in a Mac OS or Windows environment at this time." 

@mkbhanda Maybe adding a warning when someone tries to run Tern natively on MacOS would be helpful?

**Comment by nishakm** [comment]
Closing this as a note about working on MacOS was merged at commit 0c1a7b03a8f1d94044e1b7d744765b9c4d92ba82
